src/styles/blocks/cards.scss

```
@@ -0,0 +1,88 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  width: 200px;
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+
+  }
+  &__image {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+  &__title {
+    margin: 0 0 4px;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: #060b35;
+  }
+  &__text {
+    font-size: 10px;
```

Better add page break. And all over the styles.
```suggestion
  }
  
  &__title {
    margin: 0 0 4px;
    font-size: 12px;
    line-height: 18px;
    font-weight: 500;
    color: #060b35;
  }
  
  &__text {
    font-size: 10px;
```